channels/vn.m3u

```diff
@@ -1,130 +1,1075 @@
+https://vips-livecdn.fptplay.net/sda/phutho_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="PhuTho.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Phú Thọ (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sda/phutho_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="PhuYen.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Phú Yên (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/sda/phuyen_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="PhuYen.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Phú Yên (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sda/phuyen_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="PhuYen.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Phú Yên (432p)
 http://113.161.4.48:8080/phuyen/tv.m3u8
-#EXTINF:-1 tvg-id="QRT.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://uphinhnhanh.com/images/2019/01/31/IMG_20190131_191516.jpg" group-title="",QRT (404p)
+#EXTINF:-1 tvg-id="QuangBinh.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Quảng Bình (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/sda/quangbinh_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="QuangBinh.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Quảng Bình (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sda/quangbinh_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="QuangNam.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Quảng Nam (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/sdc/quangnam_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="QuangNam.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Quảng Nam (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sdc/quangnam_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="QuangNam.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Quảng Nam (404p)
 http://113.161.6.157:8081/hls-live/livepkgr/_definst_/liveevent/livestream.m3u8
-#EXTINF:-1 tvg-id="QuocHoi.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://www.quochoitv.vn/images/logo-qhtv.png" group-title="",Quốc Hội (720p)
-http://113.164.225.140:1935/live/quochoitvlive.stream_720p/playlist.m3u8
-#EXTINF:-1 tvg-id="SupremeMasterTV.vn" tvg-country="VN" tvg-language="Vietnamese;English" tvg-logo="" group-title="",Supreme Master TV (720p)
-https://lbs-us1.suprememastertv.com/720p.m3u8
```

Comment:
Working link

Reply:
#EXTINF:-1 tvg-id="QuocHoi.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://www.quochoitv.vn/images/logo-qhtv.png" group-title="",Quốc Hội (720p)
http://113.164.225.140:1935/live/quochoitvlive.stream_720p/playlist.m3u8
Existed

Reply:
#EXTINF:-1 tvg-id="SupremeMasterTV.vn" tvg-country="VN" tvg-language="Vietnamese;English" tvg-logo="" group-title="",Supreme Master TV (720p)
https://lbs-us1.suprememastertv.com/720p.m3u8

Not in Vietnam

Reply:
Where?

Reply:
> #EXTINF:-1 tvg-id="SupremeMasterTV.vn" tvg-country="VN" tvg-language="Vietnamese;English" tvg-logo="" group-title="",Supreme Master TV (720p) https://lbs-us1.suprememastertv.com/720p.m3u8
> 
> Not in Vietnam

It's in Taiwan (International Channel)